src/dom/RequestAnimationFrame.js

```diff
@@ -82,6 +82,16 @@ var RequestAnimationFrame = new Class({
          */
         this.lastTime = 0;
 
+        /**
+         * The target rate in [ms].
+         *
+         * @name Phaser.DOM.RequestAnimationFrame#target
```

Comment:
The property name feels ambiguous. It's not clear from the name (or jsdoc) that it only impacts setTimeout.

Reply:
added more docs